### PR TITLE
Vanilla entrypoint

### PR DESCRIPTION
**Pull request by mattgperry**
This PR adds an entrypoint for using Framer Motion's low-level APIs without React

```
import { animate } from "framer-motion/dom"
```

### CODE REVIEW

File: packages/framer-motion/rollup.size.config.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
+import resolve from "@rollup/plugin-node-resolve"
```
Comment (mattgperry) [inline packages/framer-motion/rollup.size.config.js:1]: This splits the distributed bundles from the bundles we generate for size. Makes `make build` faster when not measuring

**Comment by o-alexandrov** [comment]
Could you please share your plans on documenting vanilla JS usage of framer-motion?
Personally, I’d love to see layout animation working for Vue.js, as it’d bring a lot of new users to framer-motion.

**Comment by o-alexandrov** [comment]
@mattgperry thank you for the reply. I'll hope it'll turn out to be different over time, as you know yourself that if you figure out how not to rely on React, you'd improve performance dramatically. I believe this PR along with #1947 are a positive tendency increasing probability of getting a framework-agnostic `framer-motion`'s, thank you very much for the library

**Comment by mattgperry** [comment]
The React touchpoints are more about scheduling. We're barely threading the needle on having the right lifecycles to pull off layout animations, but the performance overhead is almost exclusively layout measurement and projection calculations.

**Review by hemlok — COMMENTED** [review]
👏

**Pull request merged**